x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/index.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiButton,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { useUiSetting } from '@kbn/kibana-react-plugin/public';
+import { useKibanaUrl } from '../../../hooks/use_kibana_url';
+import { AddApmData } from '../../shared/add_data_buttons/buttons';
+
+export interface ServiceTabEmptyStateProps {
+  title: string;
+  content: string;
+  imgName?: string;
+  dismissable?: boolean;
+  onDissmiss?: () => void;
+}
+
+const tryItNowButton = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.tryItNowButtonLabel', {
+    defaultMessage: 'Try it now in our demo cluster',
+  }),
+  href: 'https://ela.st/demo-apm-try-it',
+};
+
+const learnMoreLink = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.learnMoreLinkLabel', {
+    defaultMessage: 'Learn more',
+  }),
+  href: 'https://www.elastic.co/observability/application-performance-monitoring',
+};
+
+export function ServiceTabEmptyState({
+  title,
+  content,
+  imgName,
+  onDissmiss,
+}: ServiceTabEmptyStateProps) {
+  const { euiTheme } = useEuiTheme();
+  const imgFolder = `/plugins/apm/assets/service_tab_empty_state/${
```

Comment:
Should we create a constant for the base path `/plugins/apm/assets/service_tab_empty_state/`?

Reply:
Do you mean a constant shared in somewhere in the obs solution plugin?

Reply:
it could be in this file. A variable for this might make it easier to read and maintain. It's a nit, so, up to you.